src/test/java/org/jabref/gui/preferences/externalfiletypes/ExternalFileTypeItemViewModelTestData.java

```diff
@@ -0,0 +1,33 @@
+package org.jabref.gui.preferences.externalfiletypes;
+public class ExternalFileTypeItemViewModelTestData {
```

Comment:
This class is very short. Include the methods in `ExternalFileTypesTabViewModelTest` and remove the exception in the architecture tests.

I know, we had the discussion earlier. Back then, I did not really check the length of the class. In JabRef, test classes are longer and not split up into smaller classes. I don't see a reason to derivate from that style.

Reply:
@papatekken Please take action

Reply:
test classes updated